nb-configuration.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
IDE based configurations should not be committed. You can add this to .gitignore because we use Maven as a build tool. You should just import configuration from Maven to Netbeans in your local environment.